lib/ansible/modules/cloud/azure/azure_rm_aks_facts.py

```diff
@@ -18,7 +18,7 @@
 
 version_added: "2.6"
 
-short_description: Get Azure Kubernetes Service facts.
+short_description: Get Azure Kubernetes Service facts
```

Comment:
Why remove it?

Reply:
@yuwzho Here is a little hint from Ansible document, short_description should not have a trailing period/full stop([Link](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation)). Thanks a lot!